CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Add `cljr-reify-to-defrecord`
 - Add `cljr-show-changelog` so users don't have to visit github to find out what's changed after a package update.
 - Add `cljr-create-fn-from-example` to create function stub based on example usage
+- now cljr--add-test-use-declarations actually checks the file system in order to find its require for the source ns.
```

Comment:
Yes, it basically supports every kind of "convention" as the code checks if the source file (without dir and extension) is substring of the test ns.